src/scada/serialport.pas

```diff
@@ -486,7 +486,8 @@ procedure TSerialPortDriver.Read(Packet:PIOPacket);
     ReadFile(PPortHandle, Packet^.BufferToRead[Packet^.Received], Packet^.ToRead-Packet^.Received, lidos, @POverlapped);
 
     WaitForSingleObject(POverlapped.hEvent, PTimeout);
-    GetOverlappedResult(PPortHandle,POverlapped,lidos,true);
+    //IMPORTANTE , FOI MUDADO PARA "FALSE" PARA NÃO TRAVAR EM CASO DE DESLIGAMENTO DO USB
+    GetOverlappedResult(PPortHandle,POverlapped,lidos,false);  

```

Comment:
Fix to avoid windows applications to freeze when USB to serial port adapter is removed.